### ISSUE TITLE
404: v3.0.0 Not Found

### ISSUE THREAD

**Issue by eviljoe** [DavidLGoldberg/jumpy#82]
When I tried to update jumpy to v3.0.0 this morning, I got the following error message:

```
Unable to download https://www.atom.io/api/packages/jumpy/versions/3.0.0/tarball: 404 Not Found Not found
```

**Comment by blakeembrey** [comment]
![image](https://cloud.githubusercontent.com/assets/1088987/11904244/b08fe688-a573-11e5-8a90-6e6aaf27924d.png)

**Comment by mertzt89** [comment]
same here

**Comment by DavidLGoldberg** [comment]
Super sorry guys!  I pushed a new version last night, but then realized, I had an egregious bug.  

(I only noticed it when I loaded it up on some other code with long lines etc... 

I take it your old version of jumpy still works for you guys??

(I know it's been slow since 2 releases ago, been working on performance upgrade)

I ran "apm unpublish jumpy@3.0.0"  And it well..... "worked" ;) (not so much huh?) Not sure how to undo the damage!

**Comment by benfrain** [comment]
Yes, v2 still running great :+1: Look forward to v3 though :)

**Comment by jonjes** [comment]
Thanks for writing jumpy, by the way.  I use it all the time.

**Comment by DavidLGoldberg** [comment]
Thanks for the love guys!  I appreciate it! :+1:

**Comment by DavidLGoldberg** [comment]
Hey folks, I released a new version 3.0.1.  Can you guys let me know if it's working.  It should be *fast* again... *probably* faster than ever.

`apm upgrade`
or
`apm install jumpy` should fix this issue I think...if not we really should push the issue with core again... in fact, I had the foresight to do `apm publish patch` since I figured it would be stuck on 3.0.0 (hence the 3.0.1).
